awx/ui_next/src/util/useDeleteDetails.js

```diff
@@ -0,0 +1,23 @@
+export default async function getDeleteDetails(requests, count) {
+  const results = {};
+  let error = null;
+
+  if (count === 1) {
```

Comment:
what if count > 1?

Reply:
@keithjgrant  if the count is >1 we don't want this function to do anything  But, it might make more sense to do that check inside the component that calls this helper function.

If count is > 1 we just show this:

![Screen Shot 2021-02-17 at 12 18 39 PM](https://user-images.githubusercontent.com/39280967/108269187-9e6ec200-713b-11eb-8b83-4c33802cd388.png)

Reply:
In that case, I'd re-structure to: (cuts down indentation/visual complexity)

```
if (count > 1) {
  return { results, error };
}
```